services/dr-manhattan/main.py

```diff
@@ -11,41 +10,37 @@ def set_err(job, msg):
     job['data'] = []
     job['error'] = msg
 
-
 def err_check(job):
-    required = {'host', 'start_time', 'end_time'}
+    required = {'api_root', 'start_time', 'end_time'}
     if not required.issubset(job):
         set_err(job, 'Missing some required fields {}'.format(required))
 
-
 def process_message(key,job):
     err_check(job)
     if job['state'] == 'error':
         return
 
-    host = os.getenv('HOST', job['host'])
-
+    api_root = job['api_root']
```

Comment:
Why remove the Host env var?  I use that for debugging when i want to override the endpoint...
Not holding this PR up but I might want that back.